### ISSUE TITLE
Lymphangiosarcoma

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10212]
**Preferred term label:**
Lymphangiosarcoma

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0024224

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Stewart-Treves syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000004
- label: Lymphangiosarcoma
- parents: HP:0012797
- pmid: PMID:22682884
- def: A malignant neoplasm arising from the endothelial cells of the lymphatic vessels.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10212
- synonyms:
